GFX is broken on nightly

https://travis-ci.org/ozkriff/zoc/jobs/169961437

``` plain
...
cargo build --release
   Compiling core v0.0.1 (file:///home/travis/build/ozkriff/zoc/core)
   Compiling glutin v0.6.1
   Compiling tempfile v2.1.4
   Compiling wayland-window v0.2.3
   Compiling gfx_window_glutin v0.12.0
   Compiling zoc v0.0.1 (file:///home/travis/build/ozkriff/zoc)
error[E0432]: unresolved import `$crate`
  --> src/pipeline.rs:8:1
   |
8  | gfx_defines! {
   | ^ no `$crate` in the root
   |
   = note: this error originates in a macro outside of the current crate
error[E0433]: failed to resolve. Use of undeclared type or module `gfx`
  --> src/pipeline.rs:15:22
   |
15 |         basic_color: gfx::Global<[f32; 4]> = "u_Basic_color",
   |                      ^^^^^^^^^^^^^^^^^^^^^ Use of undeclared type or module `gfx`
...
```

Now (https://travis-ci.org/ozkriff/zoc/jobs/171525019) the error says:

```plain
. . .
warning: `$crate` may not be imported
  --> src/pipeline.rs:8:1
   |
8  | gfx_defines! {
   | ^
   |
   = note: `use $crate;` was erroneously allowed and will become a hard error in a future release
   = note: this error originates in a macro outside of the current crate
error[E0433]: failed to resolve. Use of undeclared type or module `gfx`
  --> src/pipeline.rs:15:22
   |
15 |         basic_color: gfx::Global<[f32; 4]> = "u_Basic_color",
   |                      ^^^^^^^^^^^^^^^^^^^^^ Use of undeclared type or module `gfx`
. . .
```

Just to offer another example...

(snip)
```
   Compiling piston2d-gfx_graphics v0.31.2
warning: `$crate` may not be imported
  --> /Users/jeff/.cargo/registry/src/github.com-1ecc6299db9ec823/piston2d-gfx_graphics-0.31.2/src/back_end.rs:30:1
   |
30 | gfx_pipeline_base!( pipe_colored {
   | ^
   |
   = note: `use $crate;` was erroneously allowed and will become a hard error in a future release
   = note: this error originates in a macro outside of the current crate

warning: `$crate` may not be imported
  --> /Users/jeff/.cargo/registry/src/github.com-1ecc6299db9ec823/piston2d-gfx_graphics-0.31.2/src/back_end.rs:39:1
   |
39 | gfx_pipeline_base!( pipe_textured {
   | ^
   |
   = note: `use $crate;` was erroneously allowed and will become a hard error in a future release
   = note: this error originates in a macro outside of the current crate

error[E0433]: failed to resolve. Use of undeclared type or module `gfx`
  --> /Users/jeff/.cargo/registry/src/github.com-1ecc6299db9ec823/piston2d-gfx_graphics-0.31.2/src/back_end.rs:31:10
   |
31 |     pos: gfx::VertexBuffer<PositionFormat>,
   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use of undeclared type or module `gfx`
```
(snip)

```
[jeff@host:~/Projects/planetkit]$ rustup run nightly rustc --version
rustc 1.14.0-nightly (3f4408347 2016-10-27)
```

--- a/src/render/src/macros/pso.rs
+++ b/src/render/src/macros/pso.rs
@@ -226,9 +226,9 @@ macro_rules! gfx_pipeline_base {
         $( $field:ident: $ty:ty, )*
     }) => {
         pub mod $module {
-            use $crate;
             #[allow(unused_imports)]
             use super::*;
+            use super::gfx;
             gfx_pipeline_inner!{ $(
                 $field: $ty,
             )*}
@@ -242,9 +242,9 @@ macro_rules! gfx_pipeline {
         $( $field:ident: $ty:ty = $value:expr, )*
     }) => {
         pub mod $module {
-            use $crate;
             #[allow(unused_imports)]
             use super::*;
+            use super::gfx;
             gfx_pipeline_inner!{ $(
                 $field: $ty,
             )*}

